dashboard/app/models/pd/workshop.rb

```diff
@@ -116,7 +111,7 @@ def subject_must_be_valid_for_course
 
   def suppress_email_subjects_must_suppress_email
     if MUST_SUPPRESS_EMAIL_SUBJECTS.include?(subject) && !suppress_email?
-      errors.add :properties, 'All academic year workshops and the Admin/Counselor - Welcome workshop must suppress email.'
+      errors.add :properties, 'All Admin/Counselor, TeacherCon, and FIT workshops must suppress email.'
```

Comment:
one danger here is that existing workshops which do not have `suppress_email` set may no longer be editable in production. Is the assumption that `suppress_email` is already set for these workshops? If not, we may have some work to do here.